DockerClientException: Could not build image: null when building from dockerfile

##### Issue Overview
When building a docker image from a dockerfile, a `DockerClientException: Could not build image: null
` is thrown, this appears to be caused by using an old version of docker-java, as it is a fixed bug in the latest versions.

##### Expected Behaviour
No exception is thrown

##### Current Behaviour
```
Caused by: com.github.dockerjava.api.exception.DockerClientException: Could not build image: null
	at com.github.dockerjava.core.command.BuildImageResultCallback.getImageId(BuildImageResultCallback.java:71)
	at com.github.dockerjava.core.command.BuildImageResultCallback.awaitImageId(BuildImageResultCallback.java:48)
	at org.arquillian.cube.docker.impl.docker.DockerClientExecutor.buildImage(DockerClientExecutor.java:738)
```


##### Steps To Reproduce

Use a dockerfile to build an image:

```
        <property name="dockerContainers">
            unit-test:
                buildImage:
                    dockerfileLocation: src/test/docker/unit-test
                    noCache: true
                    remove: true
                await:
                    strategy: sleeping
                    sleepTime: 30 s
                portBindings: [1090->1080/tcp, 9999/tcp, 8080/tcp]
                restartPolicy:
                    name: failure
                    maximumRetryCount: 1
                manual: false
```

##### Additional Information

See [https://github.com/docker-java/docker-java/issues/860](https://github.com/docker-java/docker-java/issues/860) for related docker-java issue

--- a/pom.xml
+++ b/pom.xml
@@ -66,7 +66,7 @@
     <version.arquillian_core>1.1.13.Final</version.arquillian_core>
     <version.junit>4.12</version.junit>
     <version.hamcrest>1.3</version.hamcrest>
-    <version.docker-java>3.0.6</version.docker-java>
+    <version.docker-java>3.0.14</version.docker-java>
     <version.kubernetes_client>2.6.3</version.kubernetes_client>
     <version.snakeyaml>1.18</version.snakeyaml>
     <version.shrinkwrap_resolver>3.0.0-alpha-4</version.shrinkwrap_resolver>

